src/modules/item/utils.ts

```diff
@@ -378,6 +400,10 @@ export function areEqualRepresentations(a: WearableRepresentation[], b: Wearable
   return true
 }
 
+export function areSyncedByHash(item: Item): boolean {
+  return Boolean(item.serverContentHash) && item.contentHash === item.serverContentHash
```

Comment:
This is completely fine, but the rest of the codebase is using `!!` in case you want to change it.
```suggestion
  return !!item.serverContentHash && item.contentHash === item.serverContentHash
```

again, is fine the way it is

Reply:
Changed it!